packages/inertia-react/src/useForm.js

```diff
@@ -57,7 +68,13 @@ export default function useForm(...args) {
           setHasErrors(false)
           setWasSuccessful(true)
           setRecentlySuccessful(true)
-          recentlySuccessfulTimeoutId.current = setTimeout(() => setRecentlySuccessful(false), 2000)
+          recentlySuccessfulTimeoutId.current = setTimeout(() => {
+            if (!isMounted.current) {
+              return true
```

Comment:
Curious, why are you returning `true` here?

Reply:
Well, the reasoning was: `setTimeout` did its job even if it did nothing, so `return true` :) But it could have been
```
if (isMounted.current) {
	setRecentlySuccessful(false), 2000)
}
```
instead

Reply:
Understood. You'll notice I actually wrapped a lot more code within the `isMounted.current` check, since there are other situations that could result in this warning. 👍

Reply:
Yep, saw that. I wasn't sure about those, so only fixed the thing I was able to reproduce. Thanks for your work :+1: